frontend/src/component/application/ApplicationIssues/ApplicationIssues.tsx

```diff
@@ -80,90 +80,167 @@ const StyledLink = styled(Link)(() => ({
+                length !== 1 ? 'do' : 'does'
+            } not exist in Unleash`}</Box>
+            <StyledList>
+                {features.map((feature) => (
+                    <IssueRowContainer key={feature}>
+                        <StyledListElement>{feature}</StyledListElement>
+                        <ConditionallyRender
+                            condition={hasAccess(CREATE_FEATURE)}
+                            show={
+                                <StyledLink
+                                    key={feature}
+                                    to={`/projects/default/create-toggle?name=${feature}`}
+                                >
+                                    Create feature flag
+                                </StyledLink>
+                            }
+                        />
+                    </IssueRowContainer>
+                ))}
+            </StyledList>
+        </IssueTextContainer>
+    );
 };
 
-export const ApplicationIssues = ({ issues }: IApplicationIssuesProps) => {
+const StrategiesMissing = ({ strategies }: IStrategiesMissingProps) => {
     const { hasAccess } = useContext(AccessContext);
+    const length = strategies.length;
+
+    if (length === 0) {
+        return null;
+    }
+    return (
+        <IssueTextContainer>
+            <Box>{`We detected ${length} strategy type${
+                length !== 1 ? 's' : ''
+            } defined in the SDK that ${
+                length !== 1 ? 'do' : 'does'
+            } not exist in Unleash`}</Box>
+            <StyledList>
+                {strategies.map((strategy) => (
+                    <IssueRowContainer key={strategy}>
+                        <StyledListElement>{strategy}</StyledListElement>
+                        <ConditionallyRender
+                            condition={hasAccess(CREATE_STRATEGY)}
+                            show={
+                                <StyledLink
+                                    key={strategy}
+                                    to={`/strategies/create`}
+                                >
+                                    Create strategy type
+                                </StyledLink>
+                            }
+                        />
+                    </IssueRowContainer>
+                ))}
+            </StyledList>
+        </IssueTextContainer>
+    );
+};
+
+const OutdatedSDKs = ({ sdks }: IOutdatedSDKsProps) => {
+    if (sdks.length === 0) {
+        return null;
+    }
+    return (
+        <IssueTextContainer>
+            <Box>We detected the following outdated SDKs</Box>
+            <StyledList>
+                {sdks.map((sdk) => (
+                    <IssueRowContainer key={sdk}>
+                        <StyledListElement>{sdk}</StyledListElement>
+                    </IssueRowContainer>
+                ))}
+            </StyledList>
+        </IssueTextContainer>
+    );
+};
+
+export const getApplicationIssueMode = (
```

Comment:
I'd improve the type: {applicationMode: 'success'} | {applicationMode: 'warning', issueCount: number} to make impossible states impossible